_posts/2017-03-04-r-in-action-review-1-data-structures.md

```diff
@@ -1,13 +1,18 @@
 ---
 layout: post
-title: R IN ACTION Review 1 - Creating a dataset
+title: R IN ACTION Review 1 - Data structures
 ---
 
-In [last blog][last blog] I mentioned the book "R IN ACTION". During these days,
-I read it on my way to work and home. I began from very basic knowledge (though
-I have learnt them), and got some new knowledge once again. Thus, I want to note
-and share them as weekly blogs. In this blog, I will talk about the knowledge of
-_creating a dataset_.
+During these days, I read the book "R IN ACTION" on my way to work and home.
+Reviewing very basic knowledge, feeling familiar and interesting since I got
+some new knowledge once again. Considering it could be useful and interesting in
```

Comment:
The sentence "Reviewing ... once again." consists only subordinate clauses: its main clause is missing.